models/articles.js

```diff
@@ -13,3 +13,23 @@ exports.selectArticleById = async (article_id) => {
     return Promise.reject({ status: 404, message: "Invalid article ID" });
   }
 };
+
+exports.updateArticleById = async (article_id, newArticle) => {
+  const { inc_votes } = newArticle;
+
+  if (inc_votes) {
+    // want to check if inc_votes is truthy and if so run the db.query, if not on line 32 we will reject the promise with 400
+    const result = await db.query(
+      "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *",
```

Comment:
Don't forget to end your SQL query string with a semicolon

Reply:
ahh of course, thanks Joey!